app/forms/steps/capital/property_form.rb

```diff
@@ -37,6 +37,8 @@ def persist!
       end
 
       def before_save
+        record.address = nil if is_home_address.yes?
+        record.property_owners.destroy_all if has_other_owners.no?
```

Comment:
Wondering if we add `&& (percentage_partner_owned + percentage_applicant_owned == 100)` Or similar

Reply:
Not sure if we need this rule. Happy to discuss 👍

Reply:
Agreed to leave this as is